WooCommerce/Classes/ViewRelated/Orders/Order Details/Shipping Labels/Create Shipping Label Form/Customs/ItemDetails/ShippingLabelCustomsFormItemDetails.swift

```diff
@@ -20,81 +20,136 @@ struct ShippingLabelCustomsFormItemDetails: View {
                 Text(String(format: Localization.customLineTitle, itemNumber))
-                    .font(.body)
+                    .bodyStyle()
+                Spacer()
+                Image(uiImage: .noticeImage)
+                    .foregroundColor(Color(.error))
+                    .renderedIf(viewModel.validatedItem == nil)
             }
         }, content: {
+            // Item Description
             VStack(spacing: 0) {
-                TitleAndTextFieldRow(title: Localization.descriptionTitle,
-                                     placeholder: Localization.descriptionPlaceholder,
-                                     text: $viewModel.description)
-                Divider()
-                    .padding(.leading, Constants.horizontalSpacing)
-            }
-            .padding(.horizontal, insets: safeAreaInsets)
-            .background(Color(.listForeground))
+                VStack(spacing: 0) {
+                    TitleAndTextFieldRow(title: Localization.descriptionTitle,
+                                         placeholder: Localization.descriptionPlaceholder,
+                                         text: $viewModel.description)
+                    Divider()
+                        .padding(.leading, Constants.horizontalSpacing)
+                }
+                .background(Color(.listForeground))
 
-            VStack(spacing: 0) {
-                TitleAndTextFieldRow(title: Localization.hsTariffNumberTitle,
-                                     placeholder: Localization.hsTariffNumberPlaceholder,
-                                     text: $viewModel.hsTariffNumber)
-                Divider()
-                    .padding(.leading, Constants.horizontalSpacing)
+                VStack(spacing: 0) {
+                    ValidationErrorRow(errorMessage: Localization.descriptionError)
+                    Divider()
+                        .padding(.leading, Constants.horizontalSpacing)
+                }
+                .renderedIf(!viewModel.hasValidDescription)
             }
             .padding(.horizontal, insets: safeAreaInsets)
-            .background(Color(.listForeground))
 
+            // HS Tariff Number, validation & learn more
             VStack(spacing: 0) {
-                LearnMoreRow(localizedStringWithHyperlink: Localization.learnMoreHSTariffText)
-                Divider()
-                    .padding(.leading, Constants.horizontalSpacing)
+                VStack(spacing: 0) {
+                    TitleAndTextFieldRow(title: Localization.hsTariffNumberTitle,
+                                         placeholder: Localization.hsTariffNumberPlaceholder,
+                                         text: $viewModel.hsTariffNumber,
+                                         keyboardType: .numberPad)
+                    Divider()
+                        .padding(.leading, Constants.horizontalSpacing)
+                }
+                .padding(.horizontal, insets: safeAreaInsets)
```

Comment:
I just noticed that the HS Tariff Number row is indented farther than the other rows in landscape mode, I think because this padding isn't needed:

![Simulator Screen Shot - iPhone 11 Pro - 2021-08-17 at 12 41 34](https://user-images.githubusercontent.com/8658164/129719926-feea095b-dab7-457b-a7f5-986ab49a3a33.png)

Reply:
And now I'm noticing that the background for the item details isn't edge-to-edge, so we should fix that at some point (doesn't have to be right now). Sorry for missing that in my earlier review!

Reply:
These seem to be regression issues after I restructured the stacks on the view 😅 I have fixed them in 859d4ad14116f0b7eb51dfae7be2b1b6a7090a85.

Reply:
Yeah, I was thinking it might be worth doing an overall visual check (for things like landscape, dynamic type, etc.) before we release each feature in this milestone. I've noticed regressions like this sneak in to some of my UI work and it might be easier to do a targeted review for anything like this once at the end.